### ISSUE TITLE
version number in ocamlfind

### ISSUE THREAD

**Issue by zoggy** [xavierleroy/cryptokit#26]
Hi,

cryptokit seems to have no version number given to ocamlfind. This is not very convenient to check for version constraint at configure time in other tools/libs.

**Comment by xavierleroy** [comment]
Long time no see, @zoggy!

All this is now handled by Dune, and I'm not a Dune expert.  Probably we need a `(version N.NN)` line in file `dune-project`.  Unless there's another convention to do that?  

Mentioning @snowleopard , who wrote the Dune support for Cryptokit.

**Comment by zoggy** [comment]
> Long time no see, @zoggy!

Indeed :-)

> All this is now handled by Dune, and I'm not a Dune expert. Probably we need a `(version N.NN)` line in file `dune-project`. Unless there's another convention to do that?

Dune was born while I was AFK, so I'm not yet aware of these details. I'll look at the changes. Right now I'm upgrading my codes to ocaml 4.12.0...

**Comment by snowleopard** [comment]
> Probably we need a `(version N.NN)` line in file `dune-project`.

@xavierleroy Yes, that's the simplest way forward. You will need to remember to update this line when making a new release.

> Unless there's another convention to do that?

There is a more automated solution: the [dune-release](https://github.com/ocamllabs/dune-release) tool, which automates various steps of the release process, including capturing the current version of the package and submitting the release to opam.

With this workflow, the version number is taken from the change log file (it's assumed to be the first item of the file, usually a section title), so you don't need to remember updating the `dune-project` file manually as long as you keep your change log up-to-date.
